### DISCUSSION TITLE
Can I use star constellations from Stellarium for commerical uses?

### DISCUSSION THREAD

**Discussion by CSK152004** [Stellarium/stellarium#3295] [Q&A] [answered]
Hi,

I am an 18-year-old programmer, and I am programming a quiz app about star constellations. For the app, I use screenshots from Stellarium 1.1.  Now I want to publish the app on play store for android. The game will be free and without ads.

Is it allowed to use images from stellarium for commercial use? Is it legal?

Thanks in advance.

**Comment by gzotti** [comment]
The app is free and without ads. What's your business idea then? What makes it commercial?

**Reply by CSK152004** [reply]
But its a app which will be published on play store. I only want to know if I can use the images from stellarium for it. As I said, the app will not have ads and will be free, but still it will be game app. (I really don't know much about copyright, that's why I am asking)

**Comment by gzotti** [comment]
If you study these pages for the topic, you will find the FAQ. https://github.com/Stellarium/stellarium/wiki/FAQ#can-i-use-stellarium-screenshots-in-my-bookwebsitecalendar-etc

Further, the Guide contains acknowledgements in Appendix H. Here you find license conditions for the default constellation artwork by J. Meuris. In short, it seems OK, but you need to declare where you have found them.  Note that some other skycultures may have different licenses (as also mentioned elsewhere) and may not be good for copying.

**Reply by CSK152004** [reply]
I just copying the western star const. (not the drawings, only the stars and the blue connecting lines). You are saying that its legal to use it in my game app, which will published in play store. Where do I find the credit for assets which I have to add in my game? 

Here are 2 examples of the images I use from Stellarium.

![Capella](https://github.com/Stellarium/stellarium/assets/138304352/254d61cd-2334-4c9f-8e6e-7420ccede9b7)
<img width="400" alt="Loewe" src="https://github.com/Stellarium/stellarium/assets/138304352/45464645-c8df-4406-8573-c17ddef1e59f">

**Reply by CSK152004** [reply]
That means its ok to use?

**Reply by gzotti** [reply, marked as answer]
Screenshots of the desktop version of Stellarium showing stars and connecting lines from the default "Modern" skyculture (formerly known as "Western") are OK to be used in your app. 

As sentence of credit we'd be happy to read something like:
"Constellation images created with Stellarium 23.2. https://stellarium.org"

**Reply by CSK152004** [reply]
Ok, Thank you for your detail answers. It will help me a lot. :)
